knative-serving/config-ha-services/ha-replicas-serving.adoc

```diff
@@ -1,7 +1,7 @@
 :_content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="ha-replicas-serving"]
-= High availability for Knative services
+= High availability for Knative deployments
 :context: ha-replicas-serving
 
 High availability (HA) is available by default for the Knative Serving `activator`, `autoscaler`, `autoscaler-hpa`, `controller`, `webhook`, `kourier-control`, and `kourier-gateway` components, which are configured to have two replicas each by default. You can change the number of replicas for these components by modifying the `spec.high-availability.replicas` value in the `KnativeServing` custom resource (CR).
```

Comment:
I know this part is not being changed in this PR but looking at this list of HA components, we have a few more now. This list should be updated while we're editing this file. The additional components that are HA:
* domain-mapping
* domainmapping-webhook

Reply:
Done!